scylla/src/transport/session_test.rs

```diff
@@ -388,7 +388,8 @@ async fn test_batch() {
     batch.append_statement(&format!("INSERT INTO {}.t_batch (a, b, c) VALUES (7, 11, '')", ks)[..]);
     batch.append_statement(prepared_statement.clone());
 
-    let values = ((1_i32, 2_i32, "abc"), (), (1_i32, 4_i32, "hello"));
+    let four_value: i32 = 4;
+    let values = ((1_i32, 2_i32, "abc"), (), (1_i32, &four_value, "hello"));
```

Comment:
Just a tiny nitpick/clarification: `"abc"` and `"hello"` already are references (`&str`), but their lifetime is `'static`. The problem with the code that is being reverted is that it doesn't work for references with non-static lifetimes.

No need to do anything here, I think it is obvious that the commit description talks about non-static references.